MAINTAINERS.md

```diff
@@ -49,6 +49,7 @@ Contributors may be added by existing maintainers (see [Governance document](GOV
 | JieBao Xiao    | xloya               | Xiaomi           |
 | Can Cai        | caican00            | Xiaomi           |
 | Peidian Li     | coolderli           | Xiaomi           |
+| Brandon Lu     | Lanznx           | LINE           |
```

Comment:
Hi @Lanznx can you please make your changes aligned with the above row?

Reply:
oooops @jerryshao thanks for reminding, already fixed!